sample.go

```diff
@@ -654,18 +680,17 @@ func (s *UniformSample) Sum() int64 {
 }
 
 // Update samples a new value.
-func (s *UniformSample) Updates(vv []int64)  {
-	for _,v := range vv {
+func (s *UniformSample) Updates(vv []int64) {
+	for _, v := range vv {
```

Comment:
here's a bug in my code, this should have the s.mutex locked around the calls. if we indeed use a lock...